cartography/data/jobs/cleanup/gcp_api_configs_cleanup.json

```diff
@@ -1,17 +1,17 @@
 {
     "statements": [
         {
-            "query": "MATCH (n:GCPAPIConfig)<-[:HAS_CONFIG]-(:GCPAPI)<-[:RESOURCE]-(:GCPProject{id: {GCP_PROJECT_ID}}) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
+            "query": "MATCH (n:GCPAPIConfig)<-[:HAS_CONFIG]-(:GCPAPI)<-[:RESOURCE]-(:GCPProject) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
we should not remove this parameter `GCP_PROJECT_ID` from the cleanup file. instead, we should pass this as part of the cleanup configuration object.